### PR TITLE
Daily deploys

### PR DESCRIPTION
**Pull request by benwarfield-usds**
## Changes Proposed

- deploy demo and staging daily between 9:00 and 9:30 GMT (1:00-1:30 PST/4-4:30 EST).
- remove some copy-pasta from the Azure deployment workflows

### CODE REVIEW

**Comment by benwarfield-usds** [comment]
@emmanuel-apau I am only 99% sure that what I'm deleting is no-op code—if you think it isn't, please let me know. 😄

File: .github/workflows/deployAzureBackendDemo.yml [hunk 1]
```diff
@@ -1,11 +1,9 @@
 name: Deploy Demo Backend Azure Resources
 
 on:
+  schedule:
+    - cron: '30 9 * * *' # Daily at 09:30 UTC
```
Comment (TomNUSDS) [inline .github/workflows/deployAzureBackendDemo.yml:5]: I think this is 4:30 AM ET... Just confirming we want middle-of-the-night deploys. Is the demo going down a pager offense?

Comment (benwarfield-usds) [reply .github/workflows/deployAzureBackendDemo.yml:5]: It seemed better to have it _be_ down first thing in the morning than to have it _go_ down after people had started work on other things. But yes that's one reason I wanted comments. 🙂

I don't think that either of these environments warrants a nighttime page, but demo needs to be available and up to date during the day (generally midday from what I can tell), hence this suggestion.

**Comment by timothybest-usds** [comment]
> @emmanuel-apau I am only 99% sure that what I'm deleting is no-op code—if you think it isn't, please let me know. 😄

I think the `env` in `workflow_dispatch` can be safely removed. It appears to be a section for defining inputs in the dropdown when running the action https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch

**Review by timothybest-usds — APPROVED** [review]

**Pull request merged**